e2e-tests/src/test/staking.rs

```diff
@@ -172,35 +189,42 @@ fn get_key_pairs() -> (Vec<KeyPair>, Vec<KeyPair>) {
     (stashes_accounts_key_pairs, validator_accounts_key_pairs)
 }
 
+// 1. endow stash accounts balances, controller accounts are already endowed in chainspec
```

Comment:
This is not strictly true as we're passing `a0dnet1-e2e` when generating chainspec , thus we start from the scratch (ie no bond() and validate() calls in the first place).

Reply:
Nvm, I did not ready comment properly, sorry

Reply:
Btw proper idea to add those comment here